Update index.mdx: Mention display-name property (#2460)
zmkfirmware/zmk: docs/docs/keymaps/index.mdx

1. A `bindings` property this will be a list of [behavior bindings](behaviors/index.mdx), one for each key position for the keyboard.
1. (Optional) A `sensor-bindings` property that will be a list of behavior bindings for each sensor on the keyboard. (Currently, only encoders are supported as sensor hardware, but in the future devices like trackpoints would be supported the same way)

> 1. (Optional) A `display-name` property, which is a string which will be used by certain features, such as the display for the layer status widget.

Reviewer: ```suggestion
1. (Optional) A `display-name` property that is a string used by certain features, such as the layer status display widget.
```
